blockchain/block/validator.go

```diff
@@ -36,8 +37,11 @@ func (v *validator) Validate(ctx context.Context, blk *Block) error {
 
 	v.validateActions(ctx, actions, errChan)
 	close(errChan)
+	fCtx := protocol.MustGetFeatureCtx(ctx)
 	for err := range errChan {
-		return errors.Wrap(err, "failed to validate action")
+		if !fCtx.Tolerate(err) {
+			return errors.Wrap(err, "failed to validate action")
+		}
```

Comment:
it is inexplicit to handle the error here. https://github.com/iotexproject/iotex-core/blob/5d5e5afc1e9afaddcdfaf24bb47acc00f5aed126/action/protocol/generic_validator.go#L61

Reply:
generic_validator is used by actpool and factory/statedb, I think it should keep strict checking

this error is specifically for block.validator.Validate(blk), so want to capture the error here